packages/ember-views/lib/views/text_field.js

```diff
@@ -73,7 +99,15 @@ export default Component.extend(TextSupport, {
     @type String
     @default "text"
   */
-  type: "text",
+  type: computed(function(key, value) {
+    var type = 'text';
```

Comment:
It isn't enabled yet, I could write it both ways, and use the new one if enabled though.  Slightly more work, but easier later.